ledger/ledger-utxo/build.gradle

```diff
@@ -14,6 +14,8 @@ dependencies {
     api project(':application')
     api project(':base')
     api project(':crypto')
+    api project(':cipher-suite')
```

Comment:
I am not sure if we should be referencing cipher-suite at all from any flow api modules. @alexey-kadyrov-r3 needs to clarify a bit.

Reply:
I agree, suggests that there is a type that should exist in `crypto` instead?

Reply:
This might be left over from a dependency on the `DigestService`, and can probably be removed entirely.